### ISSUE TITLE
can't find -lnghttp2

### ISSUE THREAD

**Issue by dragonstrong** [gpac/gpac#1916]
![image](https://user-images.githubusercontent.com/32926995/134191852-ff8fb1f0-1f52-4cc3-8cd9-a2c36bb3e78c.png)

## cause
the script to gererate libnghttp2 in  directory /deps_android/build/android/nghttp2.sh  inserts a command but you didn't install

`
autoreconf -i
`

## solution
first, install autoconf 

`sudo apt-get install autoconf automake libtool
`

then,  rerun the following commands:

![image](https://user-images.githubusercontent.com/32926995/134193933-63b654ab-9554-44b0-a9ba-7550a9fb8b96.png)
![image](https://user-images.githubusercontent.com/32926995/134194044-6685b1b0-f2b6-48b7-8c5d-45fdf3c7fb14.png)

**Comment by aureliendavid** [comment]
yes you need autoconf and such for this step, it could be mentioned in the docs

I added these dependencies on the wiki page in the first section https://github.com/gpac/gpac/wiki/GPAC-Build-Guide-for-Android#jvm-and-tools 

thanks for the report
